lib/mergeusertool.php

```diff
@@ -296,6 +314,14 @@ private function _merge($toid, $fromid)
                     $e->getTraceAsString() . html_writer::empty_tag('br'));
         }
 
+        if ($this->alwaysRollback) {
+            $transaction->rollback(new Exception('alwaysRollback option is set so rolling back transaction'));
+        }
+
+        if ($this->debugdb) {
```

Comment:
Absolutely - in fact rolling back the transaction throws an exception so I don't think this line was ever getting hit.